src/pycell.rs

```diff
@@ -8,9 +8,9 @@ use crate::type_object::{PyBorrowFlagLayout, PyLayout, PySizedLayout, PyTypeInfo
 use crate::types::PyAny;
 use crate::{ffi, IntoPy, PyErr, PyNativeType, PyObject, PyResult, Python};
 use std::cell::{Cell, UnsafeCell};
+use std::fmt;
```

Comment:
There's a bit of inconsistency, in other files, e.g. `types::complex`, the imports are feature-gated. Although, I think your approach here is more legible since there's just less conditionals involved.